package.json

```diff
@@ -27,6 +28,7 @@
   },
   "devDependencies": {
     "chai": "4.1.0",
-    "mocha": "3.4.2"
+    "mocha": "3.4.2",
+    "supertest": "^3.0.0"
```

Comment:
@meghanprestemon please update this dependency version to be fixed by removing the `^`, thanks!